File: 1-Iniciante/ex1020.js
// entrada
let input = require('fs').readFileSync('dev/stdin', 'utf-8'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd
let dias = parseInt(input);

// processamento
let anos = Math.floor(dias / 365);
dias -= (anos * 365);
let meses = Math.floor(dias / 30);
dias -= (meses * 30);

// saida
console.log(`${anos} ano(s)`);
console.log(`${meses} mes(es)`);
console.log(`${dias} dia(s)`);

File: 1-Iniciante/ex1020.py
#entrada
dias = int(input())

#processamento
anos = dias // 365
meses = (dias - anos * 365) // 30
dias = dias - (anos * 365 + meses * 30)

print("{:.0f} ano(s)".format(anos))
print("{:.0f} mes(es)".format(meses))
print("{:.0f} dia(s)".format(dias))
